Documentation/admin-guide/kernel-parameters.txt

```diff
@@ -2948,6 +2948,13 @@
 			[KVM,Intel] Control nested virtualization feature in
 			KVM/VMX. Default is 1 (enabled).
 
+	kvm-intel.pkvm= [KVM,Intel] If CONFIG_PKVM_INTEL is set, control whether
```

Comment:
Just noticed that early parameters are now marked with EARLY in this documentation. So `[KVM,Intel,EARLY]`?

For the record, that was introduced in v6.9, while I originally made this patch for v6.6, so I'm not guilty for missing that. :D